www/index.html

```diff
@@ -385,9 +385,9 @@ <h3 id="feedback" data-i18n="about-feedback">Feedback / helping / contributing</
                     <h3 id="credits" data-i18n="about-credits">Credits</h3>
                     <p data-i18n="about-credits-gratitude">We owe a big debt of gratitude to:
                         <ul>
-                            <li>Kelson and all the Kiwix community for welcoming us to this project;</li>
-                            <li>Wikipedia teams and contributors for helping to make knowledge available to everybody;</li>
-                            <li>All the developers of libraries and tools we used, for releasing them as Free software (see libraries and tools paragraph).</li>
+                            <li data-i18n="about-credits-gratitude-part1">Kelson and all the Kiwix community for welcoming us to this project;</li>
+                            <li data-i18n="about-credits-gratitude-part2">Wikipedia teams and contributors for helping to make knowledge available to everybody;</li>
+                            <li data-i18n="about-credits-gratitude-part3">All the developers of libraries and tools we used, for releasing them as Free software (see libraries and tools paragraph).</li>
                         </ul>
                     </p>
```

Comment:
Remove this `</p>`. You'll also need to move the indentation of the whole `<ul>...`</ul>` block back by four spaces.